deps/Versions.make

```diff
@@ -1,14 +1,14 @@
 LLVM_VER = 3.7.1
 LLVM_LIB_SUFFIX =
-PCRE_VER = 10.21
+PCRE_VER = 10.22
 DSFMT_VER = 2.2.3
 LAPACK_VER = 3.5.0
 ARPACK_VER = 3.3.0
-FFTW_VER = 3.3.4
+FFTW_VER = 3.3.5
 SUITESPARSE_VER = 4.4.5
```

Comment:
arpack and suitesparse have new versions available too but will probably hold off until the next version of suitesparse comes out, since arpack requires manually repackaging a source tarball and suitesparse finally support shared libraries upstream so we should try using that now